.changeset/red-melons-compete.md

```diff
@@ -0,0 +1,5 @@
+---
+'@braid-design-system/source.macro': patch
+---
+
+Preserve newlines in the output code
```

Comment:
Thoughts on carrying your example over to the release notes?

````suggestion
Preserve new lines in the output code

Before:
```ts
const responsiveValue = useResponsiveValue()
const isMobile = responsiveValue({
  mobile: true,
  tablet: false,
})
const isDesktopOrAbove = responsiveValue({
  mobile: false,
  desktop: true,
})
```

After:
```ts
const responsiveValue = useResponsiveValue()

const isMobile = responsiveValue({
  mobile: true,
  tablet: false,
})

const isDesktopOrAbove = responsiveValue({
  mobile: false,
  desktop: true,
})
```
````